### PR TITLE
Some improvements on main.py and added install script

### PR DESCRIPTION
**Pull request by fabiopolancoe**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,4 +5,4 @@
     name="Translation Module",
     ext_modules=cythonize("translations.pyx"),
     zip_safe=False
-)
+)
```
Comment (Sebastian-byte) [inline setup.py:8]: Here

Comment (Sebastian-byte) [inline setup.py:8]: Here

File: tranlations.pyx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    while(True):
+        command_handler(input(f"{user}@FakeOS> "))
+
+
+command_dict = {
+    "exit": sys.exit(0)
+}
+
+if __name__ == "__main__":
+    main()
```
Comment (Sebastian-byte) [inline tranlations.pyx:77]: And here

**Review by Sebastian-byte — COMMENTED** [review]
You must add a newline at the end of the files for compatibility purposes (again).

**Comment by Sebastian-byte** [comment]
Acabo de arreglar el conflicto.

**Comment by FRostri** [comment]
@fabiopolancoe  You have all ready for merge this branch with main? Or you have to do anything more?

**Comment by fabiopolancoe** [comment]
Cancelled, I will make another PR with only the install script

**Pull request closed without merging**